src/gmt_common_longoptions.h

```diff
@@ -3,7 +3,7 @@
 	 * General short-option syntax:
 	 *		-<short_option>[<short_directives>][+<short_modifiers>[<argument>]]
 	 * General long-option syntax:
-	 * 		--<long_option>[=<long_directives][+<long_modifiers>[=<argument>]]
+	 *		--<long_option>[=[<long_directives>:]<arg>][+<long_modifier1>[=<arg1>]][+<long_modifier2>[=<arg2>]]...
```

Comment:
The new developer docs have
```
--long-option[=directive][+modifier1[=arg1]][+modifier2[=arg2]][...]
```
can you explain to me why there is ```:]<arg>]``` after the long_directive here?

Reply:
Yes, I had forgotten about that.  I need to dig a bit deeper since I clearly did not provide an example when that is needed.  I think it is for options like -Qd77 which has both a directive (d) and a value (77), hence --longqname=distance:77

Reply:
Updated that in the web page as well.